### PR TITLE
[Python2&3]feat(education):Add education support to token007

### PR DESCRIPTION
**Pull request by seymourtang**
* Add education support to token007

### CODE REVIEW

**Comment by seymourtang** [comment]
@sunshinexcode

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        :param room_uuid: The room's id, must be unique.
+        :param user_uuid: The user's id, must be unique.
+        :param role: The user's role, such as 0(invisible), 1(teacher), 2(student), 3(assistant), 4(observer) etc.
+        :param expire: represented by the number of seconds elapsed since now. If, for example, you want to access the
+            Agora Service within 10 minutes after the token is generated, set expireTimestamp as 600(seconds).
+        :return: The education user room token.
+        """
+        token = AccessToken(app_id, app_certificate, expire=expire)
+
+        h = hashlib.md5()
```
Comment (sunshinexcode) [inline DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: add md5 method

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        h.update(user_uuid.encode('utf-8'))
+        char_user_id = h.hexdigest()
+        education_service = ServiceEducation(
+            room_uuid, user_uuid, char_user_id, role)
+        education_service.add_privilege(
```
Comment (sunshinexcode) [inline DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: one line

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            See Get an App Certificate.
+        :param user_uuid: The user's id, must be unique.
+        :param expire: represented by the number of seconds elapsed since now. If, for example, you want to access the
+            Agora Service within 10 minutes after the token is generated, set expireTimestamp as 600(seconds).
+        :return: The education user token.
+        """
+        token = AccessToken(app_id, app_certificate, expire=expire)
+        education_service = ServiceEducation(user_uuid=user_uuid)
+
+        education_service.add_privilege(
```
Comment (sunshinexcode) [inline DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: one line

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py (hunk 1 above)
Comment (seymourtang) [reply DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: fixed

Comment (seymourtang) [reply DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: > add md5 method

fixed

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py (hunk 2 above)
Comment (seymourtang) [reply DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: > one line

fixed

File: DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py (hunk 3 above)
Comment (seymourtang) [reply DynamicKey/AgoraDynamicKey/python/src/EducationTokenBuilder2.py]: > one line

fixed

**Pull request merged**